CitiesUIService/src/test/java/com/example/ApiDocumentation.java

```diff
@@ -54,4 +56,13 @@ public void indexExample() throws Exception {
         ;
     }
 
+    @Test
+    public void convertSwaggerToAsciiDoc() throws Exception {
```

Comment:
@mkostin please check name and structure test. I got naming for methods from recommendations in The Art of Unit Testing 2nd edition 2014 book.